src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/CloneConfigurationFixupFactory.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.ProjectSystem.VS.Properties;
+
+namespace Microsoft.VisualStudio.ProjectSystem.Properties
+{
+    /// <summary>
+    /// Interface for components that provide post-configuration cloning property fix ups.
+    /// </summary>
+    [Export(typeof(IClonePlatformFixupFactory))]
+    [AppliesTo(ProjectCapabilities.Managed)]
+    internal class CloneConfigurationFixupFactory : IClonePlatformFixupFactory
```

Comment:
It seems I need to create another interface ICloneConfigurationFixupFactory and ICloneConfigurationFixup.

In CPS, how to differentiate platform and configuration fixups ?
@davkean